### PR TITLE
yugatool: Added report header [43]

### PR DESCRIPTION
**Pull request by na6vj**
Added "Reportinfo" header to tablet report .
This includes rn-date, hostname, and yb tools version + Build date.

tablet_report_parser.pl updated to parse this new info.

### CODE REVIEW

File: tablet-report-parser/tablet-report-parser.pl [hunk 1]
```diff
@@ -282,7 +308,7 @@ sub Parse_Cluster_line{
 	}
 	print "INSERT INTO cluster(type,uuid,zone) VALUES('CLUSTER',",
 	       "'$uuid','$zone');\n";
-    if ($. > 3){
+    if ($. > 9){
```
Comment (ionthegeek) [inline tablet-report-parser/tablet-report-parser.pl:311]: Too much magic? Why 3? Why 9? Might be better to use a descriptive constant.

File: tablet-report-parser/tablet-report-parser.pl [hunk 2]
```diff
@@ -688,3 +714,4 @@ sub Parse_Tablet_line{
 }
 
 } # ----- ENd of JSON_Analyzer ---------------------------------------
```
Comment (ionthegeek) [inline tablet-report-parser/tablet-report-parser.pl:716]: Typo?

`ENd`

**Review by ionthegeek — APPROVED** [review]
Looks good to me, though I'm still learning Go so my opinion is of limited value here.

I've tested the new build and it does what it's supposed to.

File: tablet-report-parser/tablet-report-parser.pl (hunk 2 above)
Comment (na6vj) [reply tablet-report-parser/tablet-report-parser.pl:716]: fixed typo

**Comment by tvesely** [comment]
LGTM

**Pull request merged**